packages/python-packages/doc-warden/README.md

```diff
@@ -178,6 +189,21 @@ The first, `known_presence_issues`, tells warden that a presence failure detecte
 
 The `known_content_issues` parameter functions _identically_ to the `known_presence_issues` check. If a readme is listed as "already known" to have failures, the entire CI build will not be crashed by Warden.
 
+##### `package_indexing_exclusion_list` and `package_indexing_traversal_stops` Configuration
+Indexing packages is often done as part of nightly (or triggered) automation. With this being the case, sometimes `warden` may detect a PackageId that users wish to omit from the generated `packages.md` file. The Azure SDK team leverages 
+the `package_indexing_exclusion_list` array members to enable just this sort of scenario.
+
+`package_indexing_traversal_stops` is used during parse of .NET language repos _only_. This is due to how the discovery logic for readme and changelog is implemented for .NET projects. Specifically, readmes for a .csproj are often a couple directories up from their parent .csproj location!
+
+For .net, `warden` will traverse **up** one directory at a time, looking for the readme and changelog files in each traversed directory. `warden` will continue to traverse until...
```

Comment:
I assume this search behavior is because of legacy reasons? For the new structure I expect the ReadMe to be in the package directory.

Reply:
Yes. 0 of the .NET packages would have readmes if this behavior was not put in place :|